assets/base.css

```diff
@@ -742,6 +742,7 @@ details > * {
   --duration-default: 200ms;
   --duration-long: 500ms;
   --duration-extra-long: 600ms;
+  --slide-in-animation: slideIn var(--duration-extra-long) ease-out forwards;
```

Comment:
Not at all critical, but if we expect to have additional `animation` variables in the future, I wonder if we should name this `--animation-slide-in` to match the format of the `--duration-*` variables. 🤔